etrago/cluster/spatial.py

```diff
@@ -538,24 +545,26 @@ def fetch():
 
     busmap = fetch()
 
-    # TODO: Or better try/except/finally
-    if not busmap:
-        print("Busmap does not exist and will be created.\n")
-
-        cpu_cores = etrago.args["network_clustering"]["CPU_cores"]
-        if cpu_cores == "max":
-            cpu_cores = mp.cpu_count()
-        else:
-            cpu_cores = int(cpu_cores)
-
-        busmap_by_shortest_path(
-            etrago,
-            scn_name,
-            fromlvl=[110],
-            tolvl=[220, 380, 400, 450],
-            cpu_cores=cpu_cores,
+    if busmap:
```

Comment:
I guess you don't want to include this change

Reply:
I included this change intentionally. Using old eHV busmaps leads to errors in the eHV due to the different arguments for the parameter "foreign_lines": {"carrier": "AC",}

Reply:
Ok, but then I would solve it differently. Right now the busmap is only created if there is data in the table. If there is no data, you do not create the busmap and run into problems. 
And it is also not that intuitive that there is a table where busmaps are stored, but it is not used. 
I would rather change the check if the busmap from the table can be used

Reply:
Right now the busmap is always created. If there is one already, a message is shown to inform that it will be replaced. Since sometimes we know that the busmap can be reused, I will include a new argument called "reuse_busmap" in my next commit. It will be included in the "network_clustering_ehv" parameter.